open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/gui/IconChooserPanel.java

```diff
@@ -606,6 +614,24 @@ private void displayIconRecords(final List<IconRecord> recList, boolean canCance
         myLoader = new Thread(new BuildIconGridWorker(recList, canCancel));
         myLoader.start();
     }
+    
+    /**
+     * Shows resizes icons and displays loading screen until done.
+     */
+	public void IconResizer(List<IconRecord> recList,int tileWidth,boolean canCancel) 

```

Comment:
Make sure you put a javadoc for each parameter in the method.. 
`    /**
     * Shows resizes icons and displays loading screen until done.
     * @param recList 
     * @param tileWidth 
     * @param canCancel 
     */`

Reply:
The formatting is messed up on my comment but look at other methods in the class and you will see what i mean

Reply:
Also methods always start with a lowercase